### DISCUSSION TITLE
AI對醫療有什麼實質的幫助

### DISCUSSION THREAD

**Discussion by Fion-Lin** [1101-GS4719-C/Python-Community-GS4719-C#1628] [General]
As a student majoring in Biomedical Engineering, I am wondering how AI can assist human beings in terms of healthcare system? I am  not that familiar with AI but now I'm trying to learn it! Hope that it is not hard...

Link:
https://www.ibm.com/blog/the-benefits-of-ai-in-healthcare/

**Comment by lazywulf** [comment]
I can share an interesting story from one of my friends. He once trained a model to predict if one caught COVID or not using their X-ray images. The accuracy was not perfect. However, what that AI model has achieved was quite impressive.
The article did list out some possibilities. I'm looking forward to seeing this piece of tech being used to save lives.
